examples/echo.rs

```diff
@@ -25,7 +25,8 @@ async fn server(conns: usize) {
     // necessarily spawn all executors at once running symmetrical code.
     let client_handle = LocalExecutorBuilder::new()
         .pin_to_cpu(2)
-        .spawn("client", move || async move {
+        .name(String::from("client"))
```

Comment:
why can't you pass a `&str` to the name, and have the name function internally convert to a string ?